src/funfuzz/js/shared/testing-functions.js

```diff
@@ -18,8 +18,20 @@ function fuzzTestingFunctionsCtor(browser, fGlobal, fObject)
 
   function enableGCZeal()
   {
-    var level = rnd(17);
-    if (browser && level == 9) level = 0; // bug 815241
+    // As of m-c 405243:02aa9c921aed 2018-02-26
+    // https://hg.mozilla.org/mozilla-central/file/02aa9c921aed/js/src/gc/GC.cpp#l1000
+    var level = rnd(19);
+
+    // Generate the second level.
+    // ref https://hg.mozilla.org/mozilla-central/file/02aa9c921aed/js/src/gc/GC.cpp#l1001
+    var level_two = rnd(19);
+    if (level_two == 3) level_two = 0; // level_two 3 does not exist, so repurpose it
+    if (level_two == 5) level_two = 0; // level_two 5 does not exist, so repurpose it
+    if (level_two == 6) level_two = 0; // level_two 6 does not exist, so repurpose it
+
+    if (level == 3) level = 0; // level 3 does not exist, so repurpose it
+    if (level == 5) level = '"2;' + level_two + '"'; // level 5 does not exist, so repurpose it
+    if (level == 6) level = '"2;' + level_two + '"'; // level 6 does not exist, so repurpose it
```

Comment:
This looks strange, so `gczeal` can take an integer like `0`, or `"2;0"`?

Reply:
Apparently yes, see [this](https://hg.mozilla.org/mozilla-central/file/02aa9c921aed/js/src/gc/GC.cpp#l1001):

`
Specifies how zealous the garbage collector should be. Some of these modes can be set simultaneously, by passing multiple level options, e.g. "2;4" will activate both modes 2 and 4. Modes can be specified by name or number.
`